dotnet/src/Extensions/PromptTemplates.Handlebars/PromptTemplates.Handlebars.csproj

```diff
@@ -27,6 +27,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\Connectors\Connectors.OpenAI\Connectors.OpenAI.csproj" />
+    <ProjectReference Include="..\..\Functions\Functions.OpenApi\Functions.OpenApi.csproj" />
```

Comment:
Is it possible to avoid this dependency? `PromptTemplates.Handlebars` will be released as `1.0.1`, while `Functions.OpenApi` will be released as `1.0.1-alpha`. It's not critical, but if there is a chance to avoid this dependency, it would be better. Maybe this kind of functionality should be added in `Functions.OpenApi` package instead, if it's possible?

Reply:
Oh heard, I see your PR to resolve this https://github.com/microsoft/semantic-kernel/pull/4355 https://github.com/microsoft/semantic-kernel/pull/4355, will pull the changes once it's merged